src/commands/mod.rs

```diff
@@ -72,17 +75,28 @@ impl Cli {
         tokio::select! {
             res = run => res,
             _ = shutdown_signal() => {
-                eprintln!("Exiting!");
+                log::warn!("Exiting!");
                 revert_all()?;
                 Ok(())
             }
         }
     }
 
     pub fn run(self, py: pyo3::Python<'_>) -> pyo3::PyResult<()> {
+        let global = &self.global;
+        sentry::configure_scope(|scope| {
```

Comment:
Should we load credentials on every command invocation? Or configure sentry's scope upon credentials loading?

Reply:
i think i would prefer to add the user_id to the credentials and configure it on every command invocation, because as is the credentials only get loaded when we do a graphql request and there are a lot of errors that could occur before

Reply:
Let's tackle that in another pull-request?